aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -393,6 +393,13 @@ module aptos_framework::account {
         (signer, signer_cap)
     }
 
+    public entry fun transfer(source: &signer, to: address, amount: u64) {
```

Comment:
It seems strange and redundant to have Account::transfer and Coin::transfer<AptosCoin>. Is this part of the overall plan you're thinking of to reduce the number of resources for accounts that only ever transfer AptosCoin?

Reply:
Hmm... lots of folks ask the question of whether or not they have to create accounts explicitly. Usually one has in mind to create and fund an account anyway. So this is to reduce that barrier to entry.